vividus-plugin-web-app/src/main/resources/spring.xml

```diff
@@ -283,22 +248,12 @@
     </bean>
 
     <bean id="vividusWebDriverFactory" class="org.vividus.selenium.VividusWebDriverFactory">
-        <property name="remoteExecution" value="${selenium.grid.enabled}" />
-        <property name="webDriverEventListeners" ref="webDriverEventListener-EventHandlers" />
+        <constructor-arg index="0" value="${selenium.grid.enabled}"/>
+        <property name="webDriverEventListeners" ref="webDriverEventListener-EventHandlers"/>
     </bean>
 
     <bean id="webDriverManagerContext" class="org.vividus.selenium.WebDriverManagerContext" />
 
-    <bean id="storyReporter" class="org.vividus.bdd.RunContextStoryReporter">
```

Comment:
Does it make sense to move proxy steps to an extension?

Reply:
not for now, too many changes to address and to test